### PR TITLE
[WIP] Last updates

### PR DESCRIPTION
**Pull request by rodrigomha**
The following PR updates Test 10 and 11 to properly assess the effects of Dynamic Lines.
It also adds an update model for Source, that now uses an internal voltage and angle to match the solution of the power flow (via a new initialization function). Finally, new functions to print initial conditions are provided. The new printer provides bus voltages, states of dynamic devices and current for dynamic lines.

To do:
- [x] Initialization for Dynamic Lines
- [x] Update documentation example for Dynamic Lines

### CODE REVIEW

**Comment by jd-lara** [comment]
@rodrigomha this will also require the serialization of DynamicLines.

**Comment by rodrigomha** [comment]
> @rodrigomha this will also require the serialization of DynamicLines.

Yes. We need that to use the `deepcopy` of the system. We also need to update the versions of the project. I didn't update the project here, but tests are passing with PSY#master.

File: src/models/source_models.jl [hunk 1]
```diff
@@ -9,17 +9,17 @@ function mdl_source!(
 
     #Load device parameters
     bus = PSY.get_bus(device)
-    V_R = PSY.get_magnitude(bus) * cos(PSY.get_angle(bus))
-    V_I = PSY.get_magnitude(bus) * sin(PSY.get_angle(bus))
+    #V_R = PSY.get_magnitude(bus) * cos(PSY.get_angle(bus))
```
Comment (jd-lara) [inline src/models/source_models.jl]: Remove the commented code

File: src/utils/plot_utils.jl [hunk 2]
```diff
@@ -47,22 +47,62 @@ end
...
-        ix_dyn_injector =
-            PSY.get_dynamic_injector(PSY.get_component(PSY.StaticInjection, sim.system, ix))
-        if !isnothing(ix_dyn_injector)
-            println("Differential States")
-            println(ix)
+    bus_size = get_bus_count(sim.system)
+    println("Voltage Variables")
+    println("====================")
+    buses = PSY.get_components(PSY.Bus, sim.system)
+    bus_ax = [PSY.get_number(bus) for bus in buses]
```
Comment (jd-lara) [inline src/utils/plot_utils.jl]:
```suggestion
    bus_ax = sort([PSY.get_number(bus) for bus in buses])
```

File: src/utils/plot_utils.jl [hunk 3]
```diff
@@ -47,22 +47,62 @@ end
...
+    sort!(bus_ax)
+    look_up = PSY._make_ax_ref(bus_ax)
```
Comment (jd-lara) [inline src/utils/plot_utils.jl]: This is already stored in the Simulation, why create it here?

**Review by jd-lara — CHANGES_REQUESTED** [review]
There are several things in the result that retrieve code that is not necessary. Please see the comments, we already have a sorted dict with the lookup in the simulation.

**Pull request merged**